tools/tasks.py

```diff
@@ -217,10 +216,13 @@ def __init__(self, description, callable, timeout=None, **kwargs):
 
     def execute(self, fp):
         """Run the task"""
-        print("log_file: {0}. ".format(self.log_file))
+        print(("log_file: {0}. ".format(self.log_file)))
```

Comment:
Are the double parentheses needed here?

Reply:
Ah nope, thanks. Thought I'd caught those.

Reply:
Fixed :)